src/components/App/App.js

```diff
@@ -91,7 +87,7 @@ class App extends Component {
 					/>
 					<Route exact path='/MovieShowPage/:movieID'
 					render={({match}) => {
-						const movieID = match.params.movieID
+						const movieID = parseInt(match.params.movieID)
```

Comment:
This is really nice so we don't have to remember if we have to parseInt every time we need to use it as a prop.